CHANGELOG.markdown

```diff
@@ -1,3 +1,9 @@
+## Customerio 4.0.0 - July 6, 2021
```

Comment:
Not sure about the version or release date here.

Reply:
Version seems right!